talisker/logstash/talisker.filter

```diff
@@ -5,9 +5,17 @@ filter {
         negate => true
         what => "previous"
     }
+    ruby {
+      code => "
+        if event['message'].length > 20000
+            event.tag 'message truncated'
+            event['message'] = event['message'][0..20000] + '...'
```

Comment:
You have an off-by-one error here. 0..20000 is 20001 elements, so a string with 20001 elements will be marked as truncated but not actually be.